src/withDataProviders.js

```diff
@@ -54,7 +54,7 @@ export function withDataProviders(getConfig) {
     return class ComponentWithDataProviders extends React.Component {
 
       static contextTypes = {
-        dispatch: PropTypes.func.isRequired,
+        store: PropTypes.object.isRequired,
```

Comment:
When used without redux machinery, there'll be no 'store' in context. You should be prepared for such a case.